doc/thirdpartypackages/index.rst

```diff
@@ -282,6 +282,14 @@ border, and logo.
 .. image:: /_static/mpl_template_example.png
     :height: 330px
 
+figpager
+=====
+
+`figpager <https://pypi.org/project/figpager/>`_ provides customizable figure elements such as text, lines and images
```

Comment:
Please wrap at 79 char....

Reply:
Thanks! I've wrapped it below.
[https://github.com/ebenp/matplotlib/commit/898e67f6517a0744a7c39596b757d15068b44f3d](https://github.com/ebenp/matplotlib/commit/898e67f6517a0744a7c39596b757d15068b44f3d)